BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeClient.kt

```diff
@@ -335,7 +335,7 @@ open class BraintreeClient @VisibleForTesting internal constructor(
      * @param activity
      * @return [BrowserSwitchResult]
      */
-    fun deliverBrowserSwitchResult(activity: FragmentActivity): BrowserSwitchResult? {
+    open fun deliverBrowserSwitchResult(activity: FragmentActivity): BrowserSwitchResult? {
```

Comment:
Steven, what's the reason for these changes?

Reply:
Java methods are `open` by default. With the exception of Java 8, we're still trying to adhere to semantic versioning when we can. We'll be able to remove all of the `open` modifiers once in our next major version.